docs/user-guide/glossary.md

```diff
@@ -34,6 +33,22 @@ counterexample
   They all refer to an assignment of values to all of the CVL variables and
   contract storage.  See {ref}`rule-overview`.
 
+overapproximation
+underapproximation
+  Sometimes it is useful to replace a complex piece of code with something
+  simpler that is easier to reason about.  If the approximation includes all of
+  the possible behaviors of the original code (and possibly others), it is
+  called an "overapproximation"; if it does not then it is called an
+  "underapproximation".  For example, a {ref}`NONDET <view-summary>` summary is
```

Comment:
is this true even if you can reach some states outside of the actual reachable states. (i.e. the approximation reaches _some_  subset of the actual reachable states but also reaches some states _outside_ the set of actual reachable sets)?